### PR TITLE
Improve CM error management

### PR DESCRIPTION
**Pull request by soupette**
Signed-off-by: soupette <cyril.lpz@gmail.com>

### What does it do?

- Add `react-error-boundary`
- Ensure the CM's LV always have search params.

### CODE REVIEW

File: packages/strapi-helper-plugin/lib/src/components/ErrorFallback/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        <br />
+        {error.stack}
+      </details>
+    </div>
+  );
+}
+
+ErrorFallback.defaultProps = {
+  error: { message: null },
+  // resetErrorBoundary: PropTypes.func,
```
Comment (mfrachet) [inline packages/strapi-helper-plugin/lib/src/components/ErrorFallback/index.js:21]: you can remove this one if not necessary

Comment (soupette) [reply packages/strapi-helper-plugin/lib/src/components/ErrorFallback/index.js:21]: The library I have added supports such propTypes I felt it was good to comment it so when we start improving the error management we know such option is available.

WDYT of keeping the comment?

Comment (mfrachet) [reply packages/strapi-helper-plugin/lib/src/components/ErrorFallback/index.js:21]: Not a huge fan of predicting the future (personal opinion). feel free to keep it if you think it's useful 😊

Comment (soupette) [reply packages/strapi-helper-plugin/lib/src/components/ErrorFallback/index.js:21]: Let's remove it, I have added the link to the library's documentation in the component's file anyway.

**Review by mfrachet — APPROVED** [review]

**Pull request merged**